docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -137,6 +137,17 @@ This prop is available on all fields and pickers.
 
 {{"demo": "CustomFieldFormat.js"}}
 
+### Custom field placeholder
```

Comment:
It is not per-say a `format` example, but I'm putting it here because all the other places I thought off are worse.
And most of the time, I guess people will want to add localization to their placeholder just after changing the format, so I think it's the most natural place for them to easily find it.